### PR TITLE
[Spike] Update `feeProxy` examples

### PR DESCRIPTION
**Pull request by ken-centrality**
## Summary

Update the examples in `feeProxy` with a consistent way of running the examples, also add `callBatchAll` examples.

- Update README with Stackblitz and Pallet Documentation links
- Add `GUIDES.md` to instruct end-user to create and fund test account
- Fix `filterExtrinsicEvents` not correctly pick up the right data based on event names
- Update `filterExtrinsicEvents` with more complex filter
- Provide mode details to the result output

## Checklist

- [x] Add description
- [ ] Tag related issue(s)

### CODE REVIEW

**Comment by ken-centrality** [comment]
Big PR sorry @KarishmaBothara, let me know if you want me to run through it via a Slack Huddle

**Comment by KarishmaBothara** [comment]
> Big PR sorry @KarishmaBothara, let me know if you want me to run through it via a Slack Huddle

No worries.. Will look at it... if required will ask you my doubts on huddle

File: examples/substrate/use-feeProxy/src/callBatchAll.ts [hunk 1]
```diff
@@ -0,0 +1,69 @@
+import { ALICE, BOB, CHARLIE } from "@trne/utils/accounts";
+import { filterExtrinsicEvents } from "@trne/utils/filterExtrinsicEvents";
+import { formatEventData } from "@trne/utils/formatEventData";
+import { sendExtrinsic } from "@trne/utils/sendExtrinsic";
+import { withChainApi } from "@trne/utils/withChainApi";
+
+const ASTO_ASSET_ID = 17_508;
```
Comment (KarishmaBothara) [inline examples/substrate/use-feeProxy/src/callBatchAll.ts:7]: do we want to add a comment, saying ASTO_ASSET_ID = 17_508; // fee payment asset

**Review by KarishmaBothara — APPROVED** [review]
lgtm

**Pull request merged**